app/pb_kits/playbook/pb_select/_select.scss

```diff
@@ -3,7 +3,8 @@
 @import "../tokens/colors";
 
 [class^=pb_select] {
-  margin-bottom: $space_sm;
+  margin: 4px 4px $space_sm 4px;
```

Comment:
@jasoncypret we noticed that the select kit did not match the rest of the form inputs' margins, so we adjusted the margins here to match the rest of them. Thoughts on making this change?

Reply:
I think some of the PR's are working against each other, but we shouldn't be adding space on the top, left or right. We would like to enable spacing props to all kits in a future PR. For now, the plan is to only have margin-bottom defined.

Reply:
I'll remove this